gender_analysis/analysis/gender_adjective.py

```diff
@@ -367,3 +367,13 @@ def get_top_adj(full_results, num):
         top[gender] = sorted(excluded_results[gender].items(), reverse=True, key=lambda x:x[1])[0:num]
 
     return top
+
+def display_gender_adjectives(result_dict,num_to_return):
+    '''takes the results of find_gender_adj and prints out to the user the top number_to_return
+    adjectives associated with the gender searched for, sorted
+    result_dict:a dict of word-value frequency
```

Comment:
Similar to before, in order to be nicer with our style guides (and documentation tool, sadly) could you use the format `:param result_dict:` and `:return:` instead of the single colon format?